plugins/javadoc/src/main/kotlin/javadoc/JavadocPlugin.kt

```diff
@@ -31,13 +34,24 @@ class JavadocPlugin : DokkaPlugin() {
                 dokkaBasePlugin.querySingle { signatureProvider },
                 context.logger
             )
-        }
+        } applyIf { format == "javadoc" }
```

Comment:
please make desired format a const val String if we use it more than once

Reply:
ok